curvature/collector.py

```diff
@@ -53,9 +53,23 @@ def parse(self, filename, callback):
         self.join_ways()
 
         # Send our collected data to our callback function.
+        self.log("Streaming collections, each '.' is 1% complete")
+        if self.verbose:
+            total = len(self.collections)
```

Comment:
The reason I removed the streaming logging part was that I am not sure we can keep it moving forward to a streaming collector, this Is however a bit premature, let's keep this for now